src/backend/distributed/commands/multi_copy.c

```diff
@@ -602,6 +602,7 @@ CopyToExistingShards(CopyStmt *copyStatement, char *completionTag)
 static void
 CopyToNewShards(CopyStmt *copyStatement, char *completionTag)
 {
+	char *schemaName = StripSchemaName(copyStatement);
```

Comment:
Hmz, how does this work for copying via the master?

Getting an error now:

    postgres=# \COPY test.customer_reviews FROM '/home/marco/citus/customer_reviews_1998.csv' WITH (FORMAT CSV)
    ERROR:  relation "customer_reviews" does not exist